tests/Resources/ODS.robot

```diff
@@ -338,3 +338,23 @@ OpenShift Resource Component Field Should Not Be Empty
     [Arguments]    ${resource_component_field}
     Run Keyword And Continue On Failure    Should Not Be Empty    ${resource_component_field}
 
+
+Reboot Cluster Node
```

Comment:
Notes:
- I see this keyword more in tests/Resources/OCP.resource
- As I've read [in this article](https://www.linuxjournal.com/content/rebooting-magic-way), the command won't reboot the system nicely (for example, it won't  unmount or sync filesystems).  I think the keyword could be renamed to reflect that (e.g. `Force Reboot OpenShift Cluster Node`)
- Related to that, can you create new keyword `Reboot OpenShift Cluster Node` using `systemctl reboot` or `shutdown -r now` and add a new test using that to see if there is any difference?

Reply:
As we were talking, the current implementation works better to simulate a node failure